formats/application/vnd/oada/isoblue/location/1/+json/example/default.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+    _type: 'application/vnd.oada.isoblue.heartbeat.1+json',
+  
+    indexing: [
+      {
+        name: 'year-index',
+        source: 'oada.vocab.year-index',
+        value: '2019',
+      },
+      { 
+        name: 'day-index',
+        source: 'oada.vocab.day-index',
+        value: '2019-05-04',
+      },
+      {
+        name: 'hour-index',
+        source: 'oada.vocab.hour-index',
+        value: '22:00',
+      },
+    ],
+  
+    templates: {
+      '1': {
+        time: 1574107208.016281,
+        cell_ns: -79,
```

Comment:
A data point in the oada vocab is an object with a value and units.  Some units here would be nice.

Reply:
the units go in the template once you make cell_ns an object like `{ value: -79, units: 'dBm' }`

Reply:
the template would have the `{ units: 'dBm' }` part, and the data point then has the `cell_ns: { value: -79 }` part